cmd/root.go

```diff
@@ -25,6 +25,7 @@ import (
 	"io/ioutil"
 	"log"
 	"net/http"
+	URL "net/url"
```

Comment:
Please rename `url` var to something like `endpointURL` and use the standard library as-it-is.